src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReplacementServiceTests.cs

```diff
@@ -954,104 +954,136 @@ public async Task GetReplacementPlan_AllReplacementDataPresent_ReplacementValid(
                 ReleaseFileReference = replacementReleaseFileReference
             };
 
-            var originalFilterItem1 = new FilterItem
+            var originalDefaultFilterItem = new FilterItem
             {
                 Id = Guid.NewGuid(),
                 Label = "Test filter item - not changing"
             };
 
-            var originalFilterItem2 = new FilterItem
+            var originalPrimarySchoolsFilterItem = new FilterItem
             {
-                Label = "Test filter item - not changing"
+                Id = Guid.NewGuid(),
+                Label = "Primary schools"
             };
 
-            var replacementFilterItem1 = new FilterItem
+            var originalPrimaryAndSecondarySchoolsFilterItem = new FilterItem
             {
-                Label = "Test filter item - not changing"
+                Id = Guid.NewGuid(),
```

Comment:
Do you need to set this Id here? We've been avoiding adding Id's unecessarily that are set when saving the entities.

Reply:
I can see now that this is being consistent with other data being set up.